Azure/azure-cli
# [AKS] Add warnings to `az aks mesh` commands for out of support asm revision in use

**Related command**
`az aks mesh`

**Description**
This change adds warnings to the `az aks mesh` commands for an out of support asm revision in use in the cluster.

**Testing Guide**
`az aks mesh enable -- revision asm-out-of-support-revision -g rgName -n clusterName`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
